way_to_home/utils/sms_helpers.py

```diff
@@ -0,0 +1,22 @@
+"""This module provides helpers functions for sending SMS."""
+
+import nexmo
+
+from way_to_home.settings import NEXMO_API_KEY, NEXMO_API_SECRET
```

Comment:
`from django.conf import settings`
this is a better way to import settings

Reply:
Done, thank u